core_backend/tests/api/test_import_content.py

```diff
@@ -0,0 +1,265 @@
+from io import BytesIO
+from typing import Generator
+
+import pandas as pd
+import pytest
+from fastapi.testclient import TestClient
+
+
+# to be used for the DB duplicate checks later
+@pytest.fixture(scope="function")
```

Comment:
Why not use the already existing feature for `existing_content_id`?

Reply:
because we need specific values for content_title and content_text here, but that one is parameterised with values specific to that file... This is not the best, I agree.

Reply:
Then I suggest calling it something else to avoid confusion.